src/lib/posthog-client.ts

```diff
@@ -0,0 +1,26 @@
+import posthog from 'posthog-js'
+
+class PosthogClient {
```

Comment:
I made this class so that posthog only had to be initialized once and we could use the capture method wherever else we'd like.